nls/pl/strings.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * The MIT License (MIT)
+ * Copyright (c) 2017 Lance Campbell. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define */
+
+define({
+    COMMAND_NAME: "Podpowiedzi Wcięć",
+    DESCRIPTION_ENABLED: "Określa czy podpowiedzi wcięć będą pokazywane lub ukrywane.",
+    DESCRIPTION_HIDE_FIRST: "Określa czy pierwsza Podpowiedź Wcięcia będzie pokazana lub ukryta.",
+    DESCRIPTION_GUIDE_COLOR: "Kolor podpowiedzi wcięć. Może być dowolną prawidłową wartością koloru CSS.",
+    DESCRIPTION_GUIDE_STYLE: 'Ta wartość określa styl wcięć: "lity" lub "kropkowany".'
```

Comment:
@MajkelKorczak, thank you for the language contribution.

In the DESCRIPTION_GUIDE_STYLE, you need to mention that the values need to be entered as the English values "solid" or "dotted", or else people might think the Polish string values will work. Something like this:

'Ta wartość określa styl wcięć: "solid" (lity) lub "dotted" (kropkowany).'